idea/meta-storage-2/src/meta.router.ts

```diff
@@ -0,0 +1,125 @@
+import express, { Request, Response, Router } from 'express';
+import {
+  InvalidMetadataError,
+  InvalidParamsError,
+  logger,
+  MetaNotFoundError,
+  SailsIdlNotFoundError,
+} from '@gear-js/common';
+
+import { MetaService } from './service';
+
+const errorChecker = (err: Error) => {
+  if (err instanceof InvalidParamsError || err instanceof InvalidMetadataError) {
+    return 400;
+  } else if (err instanceof MetaNotFoundError || err instanceof SailsIdlNotFoundError) {
+    return 404;
+  }
+
+  return 500;
+};
+
+export class MetaRouter {
+  private router = Router({});
+
+  constructor(private metaService: MetaService) {}
+
+  public init(): void {
+    this.router.use(express.json());
+    this.router
+      .get('/meta', this.getMetaDetails.bind(this))
+      .post('/meta', this.addMetaDetails.bind(this))
+      .get('/sails', this.getSails.bind(this))
+      .post('/sails', this.addSails.bind(this));
```

Comment:
Is `.bind` necessary here?

Reply:
Yes, without the binding the express will not work properly